merlin/models/tf/features/embedding.py

```diff
@@ -249,15 +249,28 @@ def lookup_feature(self, name, val, output_sequence=False):
     def table_config(self, feature_name: str):
         return self.feature_config[feature_name].table
 
-    def embedding_table_df(
-        self, table_name: Union[str, Tags], l2_norm: bool = False, gpu: bool = True
-    ):
+    def embedding_vector(self, table_name: Union[str, Tags], l2_norm: bool = False):
```

Comment:
Wondering it we should rename this to something else. This method name might make people think it would output an individual embedding instead of the whole table.

Reply:
Makes sense. Renamed it to `get_embedding_table()`